src/components/comments/postComment.jsx

```diff
@@ -25,6 +25,12 @@ function CommentPublication(props) {
     if (tx) {
       alert("Post has been successfully created :)");
     }
+    // lens needs time to be able to retreive the new comment, 5 seconds looks like enough
+    setTimeout(() => {
+      props.query()
+      setMessage('')
+    }, 8000)
```

Comment:
This is not a good way to wait for a transaction. We need to implement this on every Lens call:  https://docs.lens.dev/docs/has-transaction-been-indexed